reflectable/lib/mirrors.dart

```diff
@@ -489,7 +489,7 @@ abstract class InstanceMirror implements ObjectMirror {
   ///     }
   ///
   /// Required capabilities: [delegate] requires the [delegateCapability].
-  dynamic delegate(Invocation invocation);
+  Object? delegate(Invocation invocation);
```

Comment:
Is it a breaking change to return `Object?` instead of `dynamic`? Should it be mentioned in the changelog?

Reply:
All methods named `invoke`/`invokeGetter`/`invokeSetter` use the return type `Object?` and `newInstance` uses `Object`, so `delegate` was the only one returning `dynamic`. Added the change to the change log. It is breaking, but the fix is local, so I think we should be OK.